specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json

```diff
@@ -4087,7 +4087,15 @@
           "description": "The node types of new cluster. Type: string (or Expression with resultType string)."
         },
         "newClusterSparkConf": {
-          "description": "a set of optional, user-specified Spark configuration key-value pairs.",
+          "description": "A set of optional, user-specified Spark configuration key-value pairs.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "object",
+            "description": "Type: string (or Expression with resultType string)."
+          }
+        },
+        "newClusterSparkEnvVars": {
+          "description": "A set of optional, user-specified Spark environment variables key-value pairs.",
```

Comment:
1. It seems we are missing an example showing creation of linked service that sets this property? is it possible to add one?
2. Any reason to use object as type of the dict's entry-value? does plain string work here?

Reply:
Hi,
Regarding (1), this property is an optional dictionary of key-value pairs specifying optional Databricks Spark environment variables.  We don't have control over what those key-values are, and are passing them right along to Databricks.  We expect customers to refer to Databricks's own API documentation to set those values.
Regarding (2), the items could be expressions, and we have a lot of other properties defined the same way too.
Can you merge this soon?
Thanks.